src/common/libutil/test/lru_cache.c

```diff
@@ -0,0 +1,87 @@
+#include <errno.h>
+
+#include "src/common/libtap/tap.h"
+#include "src/common/libutil/lru_cache.h"
+#include "src/common/libutil/xzmalloc.h"
+
+void test_basic ()
+{
+    int i;
+    lru_cache_t *lru;
+
+    lru = lru_cache_create (5);
+    ok (lru != NULL, "lru_cache_create (10)");
```

Comment:
minor: the argument should be 5 not 10 (maybe that should be a const or something?)